src/components/Logout/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { EXPRESS_OAUTH_LOGOUT_URL } from '../../configs/env';
+
+/** interface to describe props for Logout component
+ * @member {string} logoutURL the url of the logout endpoint of the Oauth server.
+ */
+export interface LogoutProps {
+  logoutURL: string;
+}
+
+export function logoutFromAuthServer(logoutURL: string) {
+  const logoutWindow: Window | null = window.open(logoutURL);
+  const timer: NodeJS.Timeout = setInterval(() => {
+    if (logoutWindow) {
+      logoutWindow.close();
+      window.location.href = EXPRESS_OAUTH_LOGOUT_URL;
+    }
+    clearInterval(timer);
+  }, 20);
+}
```

Comment:
Doc strings?

Reply:
Should this interval have a maximum number of tries? Or maybe like a timeout? (So it doesn't fire ever 20ms forever if something goes wrong?)

Reply:
@cKellyDesign I feel like we have no risk of this running on since we are opening and closing the window in a short time, then moving the execution to the server